modules/ROOT/pages/creating-and-managing-a-cluster-manually.adoc

```diff
@@ -89,10 +89,11 @@ Quorum feature is only valid for components that use Object Store.
 
 You can persistently store JDBC data in a central system that is accessible by all cluster nodes when using Mule runtime engine on-premises. Object store persistence is not currently supported for Mule applications deployed on Runtime Fabric.
 
-[TIP]
---
-Check Mule xref:hardware-and-software-requirements.adoc#mule-database-servers[Hardware and Software Requirements] for a list of supported relational database management systems.
---
+The following relational database systems are supported:
```

Comment:
should we also add any of these (from previous doc): 
* Oracle 11g
* Oracle 12c
*  IBM DB2 10
* Derby 10,

Reply:
And are the same DBs supported for Mule v4.1 and v4.2?

Reply:
Fabian responded: 
Fabián González  10:19 AM
For 1. Yes

Fabián González  10:19 AM
2. Actually they were never supported in the context of cluster